llvm/unittests/SandboxIR/SandboxIRTest.cpp

```diff
@@ -785,8 +785,9 @@ define void @foo(ptr %arg0, ptr %arg1) {
 
 TEST_F(SandboxIRTest, StoreInst) {
   parseIR(C, R"IR(
-define void @foo(i8 %val, ptr %ptr) {
+define void @foo(i8 %val, ptr %ptr, ptr %vptr) {
```

Comment:
No need to use a seaprate pointer, just use `%ptr`.

Reply:
fixed.